pkgs/applications/video/obs-studio/plugins/obs-vkcapture.nix

```diff
@@ -22,8 +23,27 @@ stdenv.mkDerivation rec {
     hash = "sha256-yaN0am24p9gC+s64Rop+jQ3952UOtZund/KttnVxP48=";
   };
 
+  cmakeFlags = lib.optionals stdenv.isi686 [
+    "-DBUILD_PLUGIN=OFF"
+  ];
+
   nativeBuildInputs = [ cmake ninja ];
-  buildInputs = [ libGL libX11 obs-studio vulkan-headers vulkan-loader wayland ];
+  buildInputs = [
+    libGL
+    libX11
+    vulkan-headers
+    vulkan-loader
+    wayland
+  ]
+  ++ lib.optionals (!stdenv.isi686) [
+    obs-studio
```

Comment:
This is because of BUILD_PLUGIN off, right?

Reply:
Correct. I think that's obvious because it's behind the same flag.